subprojects/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/PrecompiledScriptPluginIntegrationTest.kt

```diff
@@ -45,9 +46,9 @@ class PrecompiledScriptPluginIntegrationTest : AbstractPluginIntegrationTest() {
             """.trimIndent()
         )
         withPrecompiledKotlinScript(
-            "org/gradle/plugins/plugin-with-package.gradle.kts",
+            "test/gradle/plugins/plugin-with-package.gradle.kts",
             """
-            package org.gradle.plugins
+            package test.gradle.plugins
```

Comment:
❌ This shouldn't work (see https://github.com/gradle/gradle/issues/18296). We should remove the package statement.

Reply:
As discussed this will be fixed in a separate PR that will handle #18296